Tests/ProjectSpecTests/InvalidConfigsFormatTests.swift

```diff
@@ -0,0 +1,35 @@
+import ProjectSpec
+import Testing
+import TestSupport
+import PathKit
+
+struct InvalidConfigsFormatTests {
+    @Test("throws invalidConfigsFormat error for non-dictionary configs entries")
+    func testNonDictionaryConfigsEntries() throws {
+        let path = fixturePath + "invalid_configs_value_non_dict.yml"
+        let expectedError = SpecParsingError.invalidConfigsFormat(keys: ["invalid_key0", "invalid_key1"])
+
+        #expect(throws: EquatableErrorBox(expectedError)) {
+            try perform(path: path)
+        }
```

Comment:
How about getting error as the return value and checking this.
https://developer.apple.com/documentation/testing/require(throws:_:sourcelocation:performing:)-7n34r

Then you can get rid of `EquatableBox`.

Reply:
It is only available from Swift 6.1, so it can not use it.

> If you use this macro with a Swift compiler version lower than 6.1, it doesn’t return a value.

Reply:
You can use an old API. https://developer.apple.com/documentation/testing/require(_:sourcelocation:performing:throws:)